diff --git a/index.d.ts b/index.d.ts
index 7946e5a047119a072d3501d296eb21e4341e9821..04f8d6ec3b4546b7d11deebab678520b5b047551 100755
--- a/index.d.ts
+++ b/index.d.ts
@@ -22,7 +22,7 @@ import * as Koa from "koa";
 declare module "koa" {
     interface Request {
         // any declaration breaks types intellisense and type safety, keep it at least Record<string, unknown>
-        body?: Record<string, unknown>;
+        body?: any;
         rawBody: string;
     }
 }