diff --git a/pg-core/columns/enum.cjs b/pg-core/columns/enum.cjs
index e7fc534cb096262eed57cda45721787b4bea783c..4cf8200aade2ee7ae9ecf7d72fffac0b9f44584d 100644
--- a/pg-core/columns/enum.cjs
+++ b/pg-core/columns/enum.cjs
@@ -103,7 +103,7 @@ function pgEnumWithSchema(enumName, values, schema) {
 }
 function pgEnumObjectWithSchema(enumName, values, schema) {
   const enumInstance = Object.assign(
-    (name) => new PgEnumObjectColumnBuilder(name ?? "", enumInstance),
+    (name) => new PgEnumColumnBuilder(name ?? "", enumInstance),
     {
       enumName,
       enumValues: Object.values(values),
