diff --git a/polyfills/process-es6.js b/polyfills/process-es6.js
index b7205a5c508c187a096eae03a98e1ebd62854df2..c7083e516ba5e618a9c6799acf6149c2f2b51c2c 100644
--- a/polyfills/process-es6.js
+++ b/polyfills/process-es6.js
@@ -1,6 +1,8 @@
 // shim for using process in browser
 // based off https://github.com/defunctzombie/node-process/blob/master/browser.js

+import global from "./global";
+
 function defaultSetTimout() {
     throw new Error('setTimeout has not been defined');
 }