diff --git a/node_modules/jws/lib/data-stream.js b/node_modules/jws/lib/data-stream.js
index 3535d31..2ca8b87 100644
--- a/node_modules/jws/lib/data-stream.js
+++ b/node_modules/jws/lib/data-stream.js
@@ -1,6 +1,6 @@
 /*global module, process*/
 var Buffer = require('safe-buffer').Buffer;
-var Stream = require('stream');
+var Stream = require('stream').default;
 var util = require('util');
 
 function DataStream(data) {
diff --git a/node_modules/jws/lib/sign-stream.js b/node_modules/jws/lib/sign-stream.js
index 6a7ee42..fc386ba 100644
--- a/node_modules/jws/lib/sign-stream.js
+++ b/node_modules/jws/lib/sign-stream.js
@@ -2,7 +2,7 @@
 var Buffer = require('safe-buffer').Buffer;
 var DataStream = require('./data-stream');
 var jwa = require('jwa');
-var Stream = require('stream');
+var Stream = require('stream').default;
 var toString = require('./tostring');
 var util = require('util');
 
diff --git a/node_modules/jws/lib/verify-stream.js b/node_modules/jws/lib/verify-stream.js
index 39f7c73..6c7a4d9 100644
--- a/node_modules/jws/lib/verify-stream.js
+++ b/node_modules/jws/lib/verify-stream.js
@@ -2,7 +2,7 @@
 var Buffer = require('safe-buffer').Buffer;
 var DataStream = require('./data-stream');
 var jwa = require('jwa');
-var Stream = require('stream');
+var Stream = require('stream').default;
 var toString = require('./tostring');
 var util = require('util');
 var JWS_REGEX = /^[a-zA-Z0-9\-_]+?\.[a-zA-Z0-9\-_]+?\.([a-zA-Z0-9\-_]+)?$/;
