diff --git a/node_modules/quasar/dist/types/index.d.ts b/node_modules/quasar/dist/types/index.d.ts
index a24e437..b95b1d9 100644
--- a/node_modules/quasar/dist/types/index.d.ts
+++ b/node_modules/quasar/dist/types/index.d.ts
@@ -10,7 +10,7 @@ export interface AddressbarColor {
    * Sets addressbar color (for browsers that support it)
    * @param hexColor Color in hex format
    */
-  set: (hexColor: string) => void;
+  set: (hexColor?: string) => void;
 }
 
 export interface AppFullscreen {
@@ -6548,7 +6548,7 @@ export interface QPaginationProps {
    * Emitted when the component needs to change the model; Is also used by v-model
    * @param value New model value
    */
-  "onUpdate:modelValue"?: (value: string) => void;
+  "onUpdate:modelValue"?: (value: number) => void;
 }
 
 export interface QPaginationSlots {}
